Sources/Kitura/RouterRequest.swift

```diff
@@ -188,7 +188,8 @@ public class RouterRequest {
             guard let urlKeyValuePairs = body?.urlDecodedFieldValuePairs else {
                 throw Error.failedToParseRequestBody(body: body ?? "Failed to read body as String")
             }
-            return try QueryDecoder(dictionary: urlKeyValuePairs).decode(type)
+            let removedEmptyStrings = urlKeyValuePairs.filter { $0.value != "" }
```

Comment:
Would it be better to use `isEmpty` instead?

Reply:
Why don't we just change `urlDecodedFieldValuePairs` so it's `[String: String?]` instead?